roles/basic_setup_rhel/README.md

```diff
@@ -0,0 +1,93 @@
+theforeman.foreman.basic_setup_rhel
+===================================
+
+This role is an opinionated reuse of other roles in the collection, which creates a basic configuration for everything Katello needs to register and patch existing RHEL clients.
+
+That includes uploading a subscription manifest to an organization, enabling the rhel-7-server-rpms repository and creating a sync plan for it, optionally syncing it immediately, and creating an activation key `Base_RHEL_Key` to use when registering RHEL clients.
```

Comment:
Thoughts on `base_rhel_key` to simplify for users? Having to remember what is capitalized and what isn't or fat-fingering the shift key is always a source of annoyance as a user.